### PR TITLE
Article on artifact generation for deploys

### PR DESCRIPTION
**Pull request by mitchellhenke**
Draft version of article for artifact deployment. I wasn't sure how deep to go on the details, so I kept it relatively high-level.

### CODE REVIEW

File: _articles/idp-artifact-building-architecture.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+(to update this diagram, edit the [IDP artifact-building architecture][figma] file in Figma and re-export it)
+
+[figma]: https://www.figma.com/file/sam6yEo5qFMqJ29Y4jb4me/IDP-Artifact-Building-Architecture
+
+### Building and Uploading Artifacts
+
+Artifacts are built within the current IDP deploy process in the `migration` instance. The steps are:
+
+1. Download latest IDP SHA from the configured deploy branch from GitHub
+2. Skip if artifact already exists for that SHA in S3
```
Comment (zachmargolis) [inline _articles/idp-artifact-building-architecture.md:22]: I would include GitHub/source control in the diagram if possible?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/idp-artifact-building-architecture.md (hunk 1 above)
Comment (mitchellhenke) [reply _articles/idp-artifact-building-architecture.md:22]: yep, updated!

Comment (zachmargolis) [reply _articles/idp-artifact-building-architecture.md:22]: 🚢

**Pull request merged**